### DISCUSSION TITLE
someone can help me

### DISCUSSION THREAD

**Discussion by Jenongs** [qnblackcat/uYouPlus#1378] [General]
![Screenshot_5](https://github.com/qnblackcat/uYouPlus/assets/115836742/12caa4ae-c749-4b5b-9b9a-e3968292fa29)

**Comment by abacnus** [comment]
most likely your url to the youtube.ipa is not right, or the IPA itself is not right.

**Reply by ivanogames** [reply]
I think the building... thing— I'm sorry, I'm not *this* technologically inclined— has trouble working with Google Drive links. Someone on Reddit suggested using Dropbox instead AND changing the 0 in "dl=0" part of the sharing link to a 1, so "dl=1". That worked like a charm for me. The building thingy created the IPA successfully after switching to Dropbox and changing "dl=0" to "dl=1".
